website/docs/r/is_bare_metal_server.markdown

```diff
@@ -162,12 +163,12 @@ In addition to all argument reference list, you can access the following attribu
 - `href` - (String) The URL for this bare metal server
 - `id` - (String) The unique identifier for this bare metal server
 - `memory` - (Integer) The amount of memory, truncated to whole gibibytes
-- `network_interfaces` - (List) The additional network interfaces to create for the bare metal server to this bare metal server. Use `ibm_is_bare_metal_server_network_interface` resource for network interfaces.
+- `network_interfaces` - (List) The additional network interfaces to create for the bare metal server to this bare metal server. 
```

Comment:
"for the bare metal server to this bare metal server" ??

Reply:
@markrogalski, please have a look, if the newer changes make sense to you!

Reply:
Shouldn't it just be: "(List) The additional network interfaces to create for this bare metal server."

Reply:
@markrogalski Did you get chance to see the updated information!